saleor/static/dashboard-next/attributes/components/AttributeList/AttributeList.tsx

```diff
@@ -22,15 +23,33 @@ export interface AttributeListProps extends ListProps, ListActions {
 
 const useStyles = makeStyles((theme: Theme) => ({
   [theme.breakpoints.up("lg")]: {
+    colFaceted: {
+      width: 150
+    },
     colName: {
       width: "auto"
     },
+    colSearchable: {
+      width: 150
+    },
     colSlug: {
       width: 200
+    },
+    colVisible: {
+      width: 150
     }
   },
+  colFaceted: {
+    textAlign: "center"
+  },
   colName: {},
```

Comment:
empty styles

Reply:
It's for JSS to register classnames so they can be used later in responsiveness